xiplot/plugin.py

```diff
@@ -88,20 +95,23 @@ def placeholder_figure(text: str) -> Dict[str, Any]:
 def get_plugins_cached(
```

Comment:
The idea with the `plugins.py` file was to gather everything a plugin-implementer would need in a single place. Now this file mostly contains internal xiplot functions for loading plugins. Could those be moved? (e.g. to `tabs/plugins.py`)

Reply:
I don't have anything other to add at the moment, the functionality of the PR looks great!